src/components/LoginTabSelector.jsx

```diff
@@ -0,0 +1,49 @@
+import { Text, View, Image, TouchableOpacity } from 'react-native';
+import React, { Component } from 'react';
+import { containers } from '../styles/LoginScreen/Components_TabSelector';
+import { content } from '../styles/LoginScreen/Components_TabSelector';
+
+export default class LoginTabSelector extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      form: 'login',
+    };
+  }
+  render() {
+    return (
+      <View style={containers.containerSelector}>
+        <View style={containers.containerLogo}>
+          <Image source={require('../resources/Logo.png')} style={content.logo} />
+
+          <View style={containers.containerElements}>
+            <TouchableOpacity
+              style={{
+                borderBottomColor: '#7419CF',
+                borderBottomWidth: this.props.screen.state.form === 'login' ? 2 : 0,
+                width: 100,
+                alignContent: 'center',
+                alignItems: 'center',
+              }}
+              onPress={this.props.screen.handler}
+            >
+              <Text style={content.loginText}>Log In</Text>
+            </TouchableOpacity>
+            <TouchableOpacity
+              style={{
+                borderBottomColor: '#7419CF',
+                borderBottomWidth: this.props.screen.state.form !== 'login' ? 2 : 0,
+                width: 100,
+                alignContent: 'center',
+                alignItems: 'center',
+              }}
+              onPress={this.props.screen.handler}
+            >
```

Comment:
I found a bug on your code, If you are in the Log In screen and selects the Log In tab you should stay in the same page but it sends you to the Sign Up page.

Let's fix this by in your onPress props adding a condition to not change users of screen if they select their current option. 
```suggestion
            <TouchableOpacity
              style={{
                borderBottomColor: '#7419CF',
                borderBottomWidth: this.props.screen.state.form === 'login' ? 2 : 0,
                width: 100,
                alignContent: 'center',
                alignItems: 'center',
              }}
              onPress={this.props.screen.state.form != 'login' ? this.props.screen.handler : null}
            >
              <Text style={content.loginText}>Log In</Text>
            </TouchableOpacity>
            <TouchableOpacity
              style={{
                borderBottomColor: '#7419CF',
                borderBottomWidth: this.props.screen.state.form !== 'login' ? 2 : 0,
                width: 100,
                alignContent: 'center',
                alignItems: 'center',
              }}
              onPress={this.props.screen.state.form === 'login' ? this.props.screen.handler : null}
            >
```